llvm/include/llvm/DebugInfo/GSYM/CallSiteInfo.h

```diff
@@ -91,67 +85,34 @@ struct CallSiteInfoCollection {
   llvm::Error encode(FileWriter &O) const;
 };
 
-bool operator==(const CallSiteInfoCollection &LHS,
-                const CallSiteInfoCollection &RHS);
-
-bool operator==(const CallSiteInfo &LHS, const CallSiteInfo &RHS);
-
 class CallSiteInfoLoader {
 public:
   /// Constructor that initializes the CallSiteInfoLoader with necessary data
   /// structures.
   ///
   /// \param StringOffsetMap A reference to a DenseMap that maps existing string
-  /// offsets to CachedHashStringRef. \param StrTab A reference to a
-  /// StringTableBuilder used for managing looking up and creating new strings.
-  /// \param StringStorage A reference to a StringSet for storing the data for
-  /// generated strings.
-  CallSiteInfoLoader(DenseMap<uint64_t, CachedHashStringRef> &StringOffsetMap,
-                     StringTableBuilder &StrTab, StringSet<> &StringStorage)
-      : StringOffsetMap(StringOffsetMap), StrTab(StrTab),
-        StringStorage(StringStorage) {}
-
-  /// Loads call site information from a YAML file and populates the provided
-  /// FunctionInfo vector.
-  ///
+  /// offsets to CachedHashStringRef.
+  /// \param StrTab A reference to a StringTableBuilder used for managing
```

Comment:
Would you update the comment without these parameters?

Reply:
yes, remove these.